lib/ts/recipe/thirdpartyemailpassword/recipeImplementation/thirdPartyRecipeImplementation.ts

```diff
@@ -8,6 +8,10 @@ export default class RecipeImplementation implements RecipeInterface {
         this.recipeImplementation = recipeImplementation;
     }
 
+    getUsersByEmail(input: { email: string }): Promise<User[]> {
+        return this.recipeImplementation.getThirdPartyUsersByEmail(input);
```

Comment:
If we are going with my previous suggestion of creating one `getUsersByEmail` function in `ThirdPartyRecipeInterface` (it should actually be called `ThirdPartyEmailPasswordRecipeInterface` - my bad), then this function would need to be changed to filter out the users that are not third party users.